otelcolbuilder/Makefile

```diff
@@ -42,18 +42,41 @@ install-bin-linux:
 install-bin-darwin:
 	$(MAKE) _install-bin PLATFORM=darwin
 
-.PHONY: build
-build:
+ifeq (${SKIP_COMPILATION},true)
+SKIP_COMPILATION_FLAG=--skip-compilation
+else
+SKIP_COMPILATION_FLAG=
+endif
+
+.PHONY: _builder
+_builder:
 # Need to specify go path because otherwise opentelemetry-collector-builder
 # uses /usr/bin/go which on Github Actions is using preinstalled 1.15.12 by default.
 	CGO_ENABLED=${CGO_ENABLED} opentelemetry-collector-builder \
 		--go ${GO} \
 		--version ${VERSION} \
 		--config .otelcol-builder.yaml \
 		--output-path ./cmd \
+		${SKIP_COMPILATION_FLAG} \
```

Comment:
can we move to the end so there won't be 
```
--output-path ./cmd \
\
--name $(BINARY_NAME)
```
?

Reply:
Changed :) please check now